src/inference/src/dev/iasync_infer_request.cpp

```diff
@@ -201,6 +201,12 @@ void ov::IAsyncInferRequest::check_state() const {
     }
 }
 
+void ov::IAsyncInferRequest::check_cancelled_state() const {
+    std::lock_guard<std::mutex> lock{m_mutex};
+    if (m_state == InferState::CANCELLED)
```

Comment:
why `check_state` is not enough?

Reply:
check_state() will check BUSY as well, while in cpu plugin it will only check CANCELLED for several times in a single infer(), we notice at this time the state always BUSY, so check_state() will throw BUSY exception to block infer().
So cpu plugin need method to only check CANCELLED state during infer() is BUSY state.

Reply:
Who will use this method? `check_state` is called in all method, does it mean that CPU will override all default methods?